### PR TITLE
Update Step 5 Git Repositories Page (resolves #1521)

### LINKED ISSUE #1521: Update Step 5 Git Repositories Page
Under the section Summary of steps, there are two sentences need to be updated:
Clone your git \ repository and open your terminal window to it by using cd
Push the updates you just downloaded to your \ repository by using git push(info)

They symbol \ in these two sentences doesn't make sence, and it was produced by markdown syntax \<username\> in the file.

### PR DESCRIPTION
**Pull request by YangLou1**
Here is the rawgit link:
https://rawgit.com/YangLou1/YangLou1.github.io/updateStep5/#!./pages/vi/vi-github-and-repositories.md

**Checklist**

[x] Check for issue number in pull request title
[x] Are there any unneeded files in the pull request?
[x] Did they make a branch for their patch?
[x] Does the pull request actually fix the issue?
[x] Check the pull request on rawgit, does it display without any errors?
[x] Is there any merge conflicts?
[x] Make sure that people use their GitHub accounts when making commits through git

### CODE REVIEW

File: pages/vi/vi-github-and-repositories.md [hunk 1]
```diff
@@ -175,11 +175,11 @@ For more info on differences of merging vs. rebasing (and when to use which one)
 
 ## Summary of steps
 
-1. Clone your git \<username\> repository and open your terminal window to it by using ``cd``
+1. Clone your GitHub repository username.github.io and open your terminal window to it by using ``cd``
```
Comment (xyb994) [inline pages/vi/vi-github-and-repositories.md]: The original sentence does not sounds right, could you fix it?

Comment (YangLou1) [reply pages/vi/vi-github-and-repositories.md]: I already replaced the original sentence "Clone your git \ repository" with "Clone your GitHub repository username.github.io". Could you remind me what this sentence should be written or what else need to be fixed?

Comment (xyb994) [reply pages/vi/vi-github-and-repositories.md]: @YangLou1 I was referring to the latter part of the sentence, "open your terminal window to it by using cd"

Comment (YangLou1) [reply pages/vi/vi-github-and-repositories.md]: @xyb994 Thanks for reminding me. I have revised the latter part of the sentence based on the same insturction on the page.

**Pull request merged**